mpstat.c

```diff
@@ -305,7 +306,14 @@ void write_irqcpu_stats(struct stats_irqcpu *st_ic[], int ic_nr, int dis,
 				if (!strcmp(p0->irq_name, q0->irq_name) || !interval) {
 					p = st_ic[curr] + (cpu - 1) * ic_nr + j;
 					q = st_ic[prev] + (cpu - 1) * ic_nr + offset;
-					printf(" %10.2f",
+					/* Width is IRQ name + 2 for the /s */
+					colwidth=strlen(p0->irq_name)+2;
```

Comment:
Thanks for the suggestions, all good points.  Now accomodated in new pull, https://github.com/sysstat/sysstat/pull/47